common/persistence/sql/sqlplugin/mysql/visibility.go

```diff
@@ -276,8 +276,12 @@ func (mdb *db) SelectFromVisibility(
 		rows[i].StartTime = mdb.converter.FromMySQLDateTime(rows[i].StartTime)
 		rows[i].ExecutionTime = mdb.converter.FromMySQLDateTime(rows[i].ExecutionTime)
 		if rows[i].CloseTime != nil {
-			closeTime := mdb.converter.FromMySQLDateTime(*rows[i].CloseTime)
-			rows[i].CloseTime = &closeTime
+			if rows[i].CloseTime.Equal(maxMySQLDateTime) {
```

Comment:
Should this be .Greater? Also, could you add a comment explaining this

Reply:
If `close_time` is the max value, it means that the workflow close time wasn't set yet, ie., it was supposed to be NULL, but I'm setting a high value so that `ORDER BY` is done easier.

Reply:
Changed `close_time` back to NULL after discussing with @alexshtin.